app/algos/__init__.py

```diff
@@ -1,47 +1,24 @@
-from app.models import Node
-from sklearn.cluster import KMeans
-from sklearn.datasets import make_blobs
-import numpy as np
+from .kmeans import Kmeans
+from .dummy import DummyAlgo
+from app.matplot import plot_to_uri
 
-from kneed import KneeLocator
-from sklearn.datasets import make_blobs
-from sklearn.cluster import KMeans
-from sklearn.metrics import silhouette_score
-from sklearn.preprocessing import StandardScaler
+def run(method='kmeans', time_frame=1, proximity=[], diversity=[]):
```

Comment:
Does this need to be a dict of {name: val} for the attributes for proximity/diversity arrays? or do you just intend to receive the values knowing where each one is related?

Reply:
Haven't decided on that yet, but was going to work on that when I get the workflow functioning.

Reply:
Thats fine, either way works on my end so do whatever is easiest for you